MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -1563,6 +1564,13 @@ public BandwidthMetricData onLoadCompleted(String segmentUrl,
 
     private final BandwidthMetric bandwidthMetricHls = new BandwidthMetricHls();
     ArrayList<String> allowedHeaders = new ArrayList<>();
+    protected boolean debugModeOn = false;
+    protected long requestSegmentDuration = 1000;
+    protected long lastRequestSentAt = -1;
+    protected int maxNumberOfEventsPerSegmentDuration = 3;
```

Comment:
Per discussion on Monday's Standup

```suggestion
    protected int maxNumberOfEventsPerSegmentDuration = 3;
```
```suggestion
    protected int maxNumberOfEventsPerSegmentDuration = 10;
```